code/modules/power/singularity/particle_accelerator/particle.dm

```diff
@@ -29,29 +29,29 @@
 
 /obj/effect/accelerated_particle/weak
 	movement_range = 8
-	energy = 5
+	energy = 10
 	icon_state="particle0"
 	resetVariables()
 		..("energy", "movement_range")
 		movement_range = 8
-		energy = 5
+		energy = 10
 
 /obj/effect/accelerated_particle/strong
 	movement_range = 15
-	energy = 15
+	energy = 30
```

Comment:
Duplicate changes are why we use defines and these blocks of code hurt.
The relative pathing hurts
Why

MAKE DEFINE